docs/en/ingest-management/security/certificates.asciidoc

```diff
@@ -0,0 +1,216 @@
+    rxfZEqpuIjvo6fvU9ifC03WjXg1opgdEb6JqH93RHKw0PYimxhQfFcwrKxFseHUx
+    DeUNQgHkMQhDZgIfNgr9H/1X6qSU4h4LemyobKY3HDKY6pGsuBzsF4iOCtIitE9p
+    sagiWR21l1gW/lNaEW2ICKhJXbaqbE/pis45/yyPI4Q1Jd1VqZv744ejnZJnpAx9
+    mYSE5RqssMeV6Wlmu1xWljOPeerOVIKUfHY38y8GZwk7TNYAMajratG2dj+v9eAV
+    AgMBAAGjUzBRMB0GA1UdDgQWBBSCNCjkb66eVsIaa+AouwUsxU4b6zAfBgNVHSME
+    GDAWgBSCNCjkb66eVsIaa+AouwUsxU4b6zAPBgNVHRMBAf8EBTADAQH/MA0GCSqG
+    SIb3DQEBCwUAA4IBAQBVSbRObxPwYFk0nqF+THQDG/JfpAP/R6g+tagFIBkATLTu
+    zeZ6oJggWNSfgcBviTpXc6i1AT3V3iqzq9KZ5rfm9ckeJmjBd9gAcyqaeF/YpWEb
+    ZAtbxfgPLI3jK+Sn8S9fI/4djEUl6F/kARpq5ljYHt9BKlBDyL2sHymQcrDC3pTZ
+    hEOM4cDbyKHgt/rjcNhPRn/q8g3dDhBdzjlNzaCNH/kmqWpot9AwmhhfPTcf1VRc
+    gxdg0CTQvQvuceEvIYYYVGh/cIsIhV2AyiNBzV5jJw5ztQoVyWvdqn3B1YpMP8oK
+    +nadUcactH4gbsX+oXRULNC7Cdd9bp2G7sQc+aZm
+    -----END CERTIFICATE-----
+----
+--
+
+. Configure the certificates for {fleet-server}:
+
+.. If you don't already have a {fleet-server} service token, click *Fleet* and
+generate one now.
+ 
+.. From the directory where the {fleet-server} agent is installed, run the
+`enroll` (or `install`) command and specify the certificates to use. For example:
++
+--
+[source,shell]
+----
+sudo ./elastic-agent enroll \
+   --url=https://192.0.2.1:8220 \ <1>
+   --fleet-server-es=https://192.0.2.0:9200 \ <2>
+   --fleet-server-service-token=AAEBAWVsYXm0aWMvZmxlZXQtc2XydmVyL3Rva2VuLTE2MjM4OTAztDU1OTQ6dllfVW1mYnFTVjJwTC2ZQ0EtVnVZQQ \ <3>
+   --fleet-server-es-ca=/path/to/elasticsearch-ca.crt \ <4>
+   --certificate-authorities=/path/to/ca.crt \ <5>
+   --fleet-server-cert=/path/to/fleet-server.crt \ <6>
+   --fleet-server-cert-key=/path/to/fleet-server.key <7>
+----
+
+<1> {fleet-server} URL
+<2> {es} URL
+<3> CA certificate to use to connect to {es}
+<4> Service token to use to communicate with {es}
+<5> CA certificate to use to connect to {fleet-server}
+<6> Certificate to use for the exposed {fleet-server} HTTPS endpoint
+<7> Private key to use for the exposed {fleet-server} HTTPS endpoint
+--
++
+.What happens if you enroll {fleet-server} without specifying certificates?
+****
+
+If the certificates are managed by your organization and installed at the system
+level, they will be used to encrypt traffic between {agent}s, {fleet-server},
+and {es}.
+
+If system-level certificates don't exist, {fleet-server} automatically generates
+self-signed certificates. Traffic between {fleet-server} and {agent}s over
+HTTPS is encrypted, but the certificate chain cannot be verified. Any {agent}s
+enrolling in {fleet-server} will need to pass the `--insecure` flag to
+acknowledge that the certificate chain is not verified.
+
+Allowing {fleet-server} to generate self-signed certificates is useful to get
+things running for development, but not recommended in a production environment.
+****
+
+.. Start the {fleet-server} agent.
+
+. Next, enroll (or install) your {agent}s into {fleet}. {agent}s connecting to a
```

Comment:
This step needs to be expanded (maybe include a link to the docs about installing agents) rather than duplicating the info.